### PR TITLE
Update methods to match api endpoint changes

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

File: src/user.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const { method } = require('./http');
+const mapValues = require('lodash/mapValues');
+
+
+const getPermissions = (options = {}) => ({
+  method: 'GET',
+  url: `/user/permissions/`,
```
Comment (justinvdm) [inline src/user.js:7]: If we are going to be changing removing of permissions to be possible via `DELETE /permissions/<id>`, then we need to move the current `/permissions/` method that obtains a user's permissions from a token to somewhere else. I vote for `GET /user/permissions/`, but up for other suggestions.

**Comment by justinvdm** [comment]
@rudigiesler ready for review

**Comment by justinvdm** [comment]
@rudigiesler reverted the permissions change, should be ready for review

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**